dev/Common/IsWindowsVersion.h

```diff
@@ -8,42 +8,32 @@
 
 namespace WindowsVersion
 {
-VERSIONHELPERAPI IsWindowsVersionOrGreaterEx(
-    const WORD majorVersion,
-    const WORD minorVersion,
-    const WORD servicePackMajor,
-    const WORD buildNumber)
+inline bool IsExportPresent(
+    PCWSTR filename,
+    PCSTR functionName)
 {
-    OSVERSIONINFOEXW osvi{ sizeof(osvi) };
-    osvi.dwMajorVersion = majorVersion;
-    osvi.dwMinorVersion = minorVersion;
-    osvi.wServicePackMajor = servicePackMajor;
-    osvi.dwBuildNumber = buildNumber;
-
-    const DWORDLONG c_conditionMask{
-        VerSetConditionMask(
-            VerSetConditionMask(
-                VerSetConditionMask(
-                    VerSetConditionMask(
-                        0, VER_MAJORVERSION, VER_GREATER_EQUAL),
-                    VER_MINORVERSION, VER_GREATER_EQUAL),
-                VER_SERVICEPACKMAJOR, VER_GREATER_EQUAL),
-            VER_BUILDNUMBER, VER_GREATER_EQUAL)
-    };
-
-    return VerifyVersionInfoW(&osvi, VER_MAJORVERSION | VER_MINORVERSION | VER_SERVICEPACKMAJOR | VER_BUILDNUMBER, c_conditionMask) != FALSE;
+    wil::unique_hmodule dll{ LoadLibraryExW(filename, nullptr, LOAD_WITH_ALTERED_SEARCH_PATH) };
```

Comment:
>LOAD_WITH_ALTERED_SEARCH_PATH [](http://example.com/codeflow?start=63&length=29)

according to MSDN: If this value is used and lpFileName specifies a relative path, the behavior is undefined.

Reply:
Good point. And it's unnecessary; flags were used in an early iteration and I forgot to scrub it. Fixed